aim/sdk/run.py

```diff
@@ -645,9 +645,9 @@ def _read_sequence_info(self, ctx, name):
         seq_info = self.sequence_info[sequence_selector]
 
         assert not seq_info.initialized
-        seq_info.val_view = self.series_run_tree.subtree(sequence_selector).array('val').allocate()
-        seq_info.epoch_view = self.series_run_tree.subtree(sequence_selector).array('epoch').allocate()
-        seq_info.time_view = self.series_run_tree.subtree(sequence_selector).array('time').allocate()
+        seq_info.val_view = self.series_run_tree.subtree(sequence_selector).array('val', dtype='double').allocate()
```

Comment:
this function is meant for generic sequences. we cannot be sure that `dtype='double'` is the correct choice.
though looking into the implementation of `TreeArrayView` it is only used to convert to numpy...

Reply:
This is my fault. Yes, we have to figure out how to treat series / sequences as typed arrays.
Will roll back this one.